### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by astj**
- Use SVG badge for coverall, which matches better with other (SVG) badges
- Import without alias in usage section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,20 +14,20 @@ mackerel-client-go is a Go client library for [mackerel.io API](https://mackerel
 # Usage
 
 ```go
-import mkr "github.com/mackerelio/mackerel-client-go"
+import "github.com/mackerelio/mackerel-client-go"
 ```
 
 ```go
-client = mkr.NewClient("<Put your API key>")
+client = mackerel.NewClient("<Put your API key>")
```
Comment (itchyny) [inline README.md]: User may use `:=` for defining the client.

Comment (astj) [reply README.md]: Fixed at 
https://github.com/mackerelio/mackerel-client-go/pull/74/commits/5ea755dd5c2d3eb6812289b067c9703fd41afa5b .

**Comment by itchyny** [comment]
LGTM

**Comment by astj** [comment]
Thanks.

**Pull request merged**